dpe/src/commands/derive_context.rs

```diff
@@ -36,6 +36,9 @@ bitflags! {
         const INPUT_ALLOW_CA = 1u32 << 26;
         const INPUT_ALLOW_X509 = 1u32 << 25;
         const RECURSIVE = 1u32 << 24;
+        const EXPORT_CDI = 1u32 << 23;
+        const CREATE_CERTIFICATE = 1u32 << 22;
+        const RETURN_CERTIFICATE = 1u32 << 21;
```

Comment:
In the spec, I think there is only `CREATE_CERTIFICATE` flag, not `RETURN_CERTIFICATE`. Is there a reason we need both?

Reply:
Looking at "TCG DICE Protection Environment Specification v1.0 FINAL"

It says:

> return-certificate: Indicates whether a DPE MUST return the generated certificate when create-certificate is true. If true, the certificate is returned in the certificate output argument. If create-certificate is false, this argument is ignored. If omitted, the default value of this argument is false.

I think `return-certificate` is redundant personally. Did I look at the wrong spec?

Reply:
Oh I see, the profile specifies this can be omitted. I think the profile specification will need to be updated to default to `true`.

Reply:
Thanks!